jest.config.js

```diff
@@ -1,8 +1,16 @@
 // For a detailed explanation regarding each configuration property, visit:
 // https://jestjs.io/docs/en/configuration.html
 
-module.exports = {
-	preset: "ts-jest",
+export default {
+	preset: "ts-jest/presets/default-esm",
+	globals: {
+		"ts-jest": {
+			useESM: true,
+		},
+	},
+	moduleNameMapper: {
+		"(.*)\\.js$": ["$1"],
+	},
```

Comment:
I had to tell the `@next` version of `ts-jest` and `jest` to strip off the `.js` from the imports since they technically didn't exist and `ts-jest` doesn't resolve the ghosts like the eslint plugin import resolver for TypeScript.